cafe/drivers/unittest/decorators.py

```diff
@@ -177,6 +178,23 @@ def decorator(cls):
                 class_name_new = "{0}_{1}".format(class_name, dataset.name)
                 new_class = type(class_name_new, (cls,), dataset.data)
                 new_class.__module__ = cls.__module__
+
```

Comment:
This should probably be wrapped in a guard statement checking if there are any tags or decorators for the dataset. If there's not, this loop will still run and copy the test anyway. It doesn't hurt anything, but it's not necessary (credit goes to @robe6162 for catching that!)

Reply:
Since we need line 199 in both situations, I went with a nested conditional over a guard clause. I could change this to:
```py
if not (dataset.metadata["tags"] or dataset.metadata["decorators"]):
    setattr(module, class_name_new, new_class)
    continue
```
if you or @robe6162 prefer, but I didn't want to repeat that line.